src/GovUk.Education.ExploreEducationStatistics.Content.Services/ReleaseFileService.cs

```diff
@@ -18,45 +19,40 @@
 using GovUk.Education.ExploreEducationStatistics.Content.Services.Interfaces;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.EntityFrameworkCore;
-using Microsoft.Extensions.Logging;
 using static GovUk.Education.ExploreEducationStatistics.Common.BlobContainers;
+using File = System.IO.File;
 
 namespace GovUk.Education.ExploreEducationStatistics.Content.Services
 {
     public class ReleaseFileService : IReleaseFileService
     {
-        private static readonly FileType[] DownloadableFileTypes = {
-            FileType.Ancillary,
-            FileType.Data,
-        };
+        /// How long the all files zip should be
```

Comment:
After a release is published, would of thought that the files should never change if nothing is editable without creating an amendment, so I think we could have got away with this either being much longer or unlimited? Nice to have this in  though so that the zip file can easily evolve over time like the addition of data guidance without us touching the files.

Reply:
Yeh I did think about that, but decided against as I imagined DfE would want the file to be updated fairly soon after a deployment updating the zip file goes out. Otherwise, they might be a little confused why the update hasn't worked. Of course, there is still a (current) timeframe of up to an hour where the file might be stale, but feels acceptable. Could potentially bump it up even higher if DfE are aware of the tradeoff though?

The other option I thought about was to add a new metadata field called `Version` or something to each zip blob. This would just be some hard-coded string from `ReleaseFileService` that we could bump up whenever there's a change to the file. My man issue is it could be accidentally missed out whenever someone makes a change to the zip and ends up needing another deployment to fix.

What do you think?

Reply:
Ah ok, it seems fine as it is then! I expect that any updates being deployed that can change the content of the zip file will be fairly infrequent, and 1 hour is a short-ish amount of time to wait if there is a change.

I think the version field is a good idea but seems overkill for now and like you say, we might forget to update it. 🙂